core/src/main/codegen/templates/Parser.jj

```diff
@@ -1579,15 +1579,38 @@ List<SqlNode> SelectList() :
 SqlNode SelectItem() :
 {
     SqlNode e;
-    final SqlIdentifier id;
+    SqlIdentifier id;
+    final List<SqlNode> ids = new ArrayList();
+    final Span s = span();
 }
 {
     e = SelectExpression()
     [
         [ <AS> ]
-        id = SimpleIdentifier() {
-            e = SqlStdOperatorTable.AS.createCall(span().end(e), e, id);
-        }
+        (
+            (
+              id = SimpleIdentifier()
+              {
+                  e = SqlStdOperatorTable.AS.createCall(s.end(e), e, id);
+              }
+            )
+            |
+            (
+              <LPAREN>
+                 id = SimpleIdentifier()
+                 {
+                    ids.add(id);
+                 }
+                 ( <COMMA> id = SimpleIdentifier() { ids.add(id);} )*
+              <RPAREN>
+              {
+                  if (!this.conformance.allowSelectTableFunction()) {
+                    throw new ParseException(RESOURCE.notAllowTableFunctionInSelect().str());
+                  }
```

Comment:
Please do not throw ParseException directly, instead use `SqlUtil.newContextException` to have more detailed error message.

Reply:
Thanks for you suggestions @danny0405 .

Reply:
I have pushed the change, but this page has not refresh to the latest code.

Reply:
Thanks, the new change looks fine ~